scripts/ci_build.cmd

```diff
@@ -3,6 +3,23 @@
 REM Copyright (c) .NET Foundation and contributors. All rights reserved.
 REM Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-CALL %~dp0..\build.cmd %*
+:main
```

Comment:
The point of this file being so slim was so that the ci_build did not do anything significantly different than the non-ci build.

Any reason not to put this in scripts/package/package.ps1 with other packaging logic?